lib/services/presets.go

```diff
@@ -271,6 +330,30 @@ func AddDefaultAllowConditions(role types.Role) (types.Role, error) {
 		}
 	}
 
+	if role.GetName() == teleport.PresetAccessRoleName {
+		allowARC := role.GetAccessRequestConditions(types.Allow)
+		if len(allowARC.Annotations) == 0 &&
+			len(allowARC.ClaimsToRoles) == 0 &&
+			len(allowARC.Roles) == 0 &&
+			len(allowARC.SearchAsRoles) == 0 &&
+			len(allowARC.SuggestedReviewers) == 0 &&
+			len(allowARC.Thresholds) == 0 {
```

Comment:
Would it make sense to have `IsEmpty()` method on access request conditions or something? Otherwise we may add a new field and forget to update here.

Same for access review conditions below.